run_dir/static/js/flowcell_samples.js

```diff
@@ -185,11 +190,12 @@ $.getJSON("/api/v1/flowcell_info2/"+flowcell, function(data) {
                     }
                     // Make count nice and work out percentage
                     var count = parseInt(data['undetermined'][lid][ordered_keys[ud]]);
-                    var percentage = ((count/total)*100).toFixed(2);
+                    var percentage = (100 * count/total_undetermined_claster_number).toFixed(2);
```

Comment:
I'm confused though - the next two lines are not new, so has it always done this?